### DISCUSSION TITLE
Generating PDF report on Linux

### DISCUSSION THREAD

**Discussion by Munchic** [vdemichev/DiaNN#383] [Q&A]
Hello, I'm using the Linux command line version of DIA-NN and couldn't find the flag for generating the PDF statistic reports that the GUI version has. May I know how I can do that? Thank you!

**Comment by vdemichev** [comment]
Hi, sorry, unfortunately this is currently impossible on Linux.

**Reply by Munchic** [reply]
Hi Vadim, thanks! I have everything already run on Linux and just need to do the plots, can I do that in Windows somehow?

**Reply by vdemichev** [reply]
By manually calling PDF plotter - yes. DIA-NN GUI prints the respective command at the end of the analysis.

**Reply by Munchic** [reply]
Thank you, I will give that a try!
